core/templates/dev/head/css/oppia.css

```diff
@@ -2873,3 +2873,514 @@ md-card.preview-conversation-skin-supplemental-card {
   color: red;
   margin-bottom: 10px;
 }
+
+.conversation-skin-progress-dots {
```

Comment:
Do you think it's OK to keep the .conversation-skin-* stuff with the conversation_skin directive for now, instead of moving it to the general oppia.css file? We're actually trying to split up the oppia.css file into component-specific CSS, since it's getting quite unwieldy. The advantage of this approach is also that unnecessary CSS won't get included when the component is not part of the page.